tests/clone_test.go

```diff
@@ -653,6 +653,48 @@ var _ = Describe("[Serial][sig-compute]VirtualMachineClone Tests", Serial, decor
 					Expect(libinstancetype.EnsureControllerRevisionObjectsEqual(sourceVM.Spec.Preference.RevisionName, targetVM.Spec.Preference.RevisionName, virtClient)).To(BeTrue(), "source and target preference controller revisions are expected to be equal")
 				})
 			})
+
+			It("double cloning: clone target as a clone source", func() {
+				addAnnotationAndLabelFilters := func(vmClone *clonev1alpha1.VirtualMachineClone) {
+					filters := []string{"somekey/*"}
+					vmClone.Spec.LabelFilters = filters
+					vmClone.Spec.AnnotationFilters = filters
+				}
+				generateCloneFromVMWithParams := func(sourceVM *virtv1.VirtualMachine, targetVMName string) *clonev1alpha1.VirtualMachineClone {
+					vmclone := generateCloneFromVMWithParams(sourceVM, targetVMName)
```

Comment:
This is so confusing!! **Please** consider naming the 2 functions differently!

Reply:
Done.

Although... I have to say I'm not sure I agree :) I actually like the way of expressing "in this scope, this function operates a bit differently. Do the same as above but with added logic". But yeah, I can see how it's confusing for some.